File: CPP/Program/Garage.cpp
// Psrkinglot parkir;

#pragma once
// Memasukan library yang digunakan
#include <iostream>
#include <string>
#include "Parkinglot.cpp"

// Deklarasi namespace
using namespace std;

// Deklarasi kelas
class Garage
{
private:
    // Atribut Privat
    string nama;
    string luas;
    string daftar;
    // Composite dengan Parkinglot
    Parkinglot parkir;

public:
    // Constructor
    Garage()
    {

    }

    //Constructor dengan Parameter
    Garage(string nama, string luas, string daftar, Parkinglot parkir)
    {
        this->nama = nama;
        this->luas = luas;
        this->daftar = daftar;
        this->parkir = parkir;
    }

    // Setter dan Getter
    void setNama(string nama)
    {
        this->nama = nama;
    }

    void setLuas(string luas)
    {
        this->luas = luas;
    }

    void setDaftar(string daftar)
    {
        this->daftar = daftar;
    }

    void setParkir(Parkinglot parkir)
    {
        this->parkir = parkir;
    }

    string getNama()
    {
        return this->nama;
    }

    string getLuas()
    {
        return this->luas;
    }

    string getDaftar()
    {
        return this->daftar;
    }

    Parkinglot getParkir()
    {
        return  this->parkir;
    }

    // Destructor
    ~Garage()
    {

    }
};

// Garage::Garage(/* args */)
// {
// }

// Garage::~Garage()
// {
// }

File: Python/Program/Garage.py
## Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP2 dalam mata kuliah 
## Desain Pemorograman Berbasis Objek untuk keberkahanNya maka saya tidak 
## melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.

## Membuat class
class Garage:
    ## Atribut privat
    nama=""
    luas=""
    daftar=""
    parkir=None ## Composite
    
    ## Constructor
    ## Atribut Publik
    def __init__(self, nama="", luas="", daftar="", parkir=None):
        self.nama = nama
        self.luas = luas
        self.daftar = daftar
        self.parkir = parkir

    ## Getter dan Setter
    def setNama(self, nama):
        self.nama = nama

    def setLuas(self, luas):
        self.luas = luas

    def setDaftar(self, daftar):
        self.daftar = daftar

    def setParkir(self, parkir):
        self.parkir = parkir

    def getNama(self):
        return self.nama

    def getLuas(self):
        return self.luas

    def getDaftar(self):
        return self.daftar

    def getParkir(self):
        return self.parkir